pkg/utils/util.go

```diff
@@ -0,0 +1,22 @@
+package utils
```

Comment:
[Non Blocking][Suggestion]
Might make sense to rename the file to something like `namespace` or similar

Reply:
That's a much better name, thank you!

Reply:
> utils isn't an ideal package name for the reasons stated [here](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common).
namespace may cause import conflicts in files that use this utility and https://github.com/celestiaorg/nmt/blob/master/namespace/doc.go

second this comment.